### ISSUE TITLE
Local .xml file

### ISSUE THREAD

**Issue by jasperjorna** [Ayima/sitemap-visualization-tool#4]
Hi there,

Is is possible to pass in a local .xml file instead of a URL?

**Comment by zazencodes** [comment]
Not exactly. However if you can extract the URLs from the XML file yourself then it's possible.

You would need to put the URLs into a file named `sitemap_urls.dat` where there is one URL per line and no header.

Then, ensuring you have that file in the working directory, you can go ahead and run e.g.
```
python categorize_urls.py --depth 3   
python visualize_urls.py --depth 3 --title "My Sitemap" --size "20"   
```
